### PR TITLE
Feature/bliss 381/update sdk to v6.2.0 ios

### PR DESCRIPTION
**Pull request by hiteshjain29**
feat(public api): Add PII masking and form removal for ios

### CODE REVIEW

**Comment by kioli** [comment]
The PR is meant to be merged to branch `BLISS-362` but it's based on branch `BLISS-380` so it should be also targeted to it

File: ios/ReactNativeUsabilla/UsabillaInterface.swift [hunk 1]
```diff
@@ -83,6 +83,25 @@ class UsabillaInterface: NSObject {
     func takeScreenshot(view: UIView) -> UIImage {
         return Usabilla.takeScreenshot(view)!
     }
+    
+    @objc(dismiss)
+    func dismiss() {
+        let _ = Usabilla.dismiss()
```
Comment (kioli) [inline ios/ReactNativeUsabilla/UsabillaInterface.swift]: Does it need to be assigned a value to a variable?
If so should we return it?

File: ios/ReactNativeUsabilla/UsabillaInterface.swift [hunk 2]
```diff
@@ -83,6 +83,25 @@ class UsabillaInterface: NSObject {
...
+    }
+    
+    @objc(getDefaultDataMasks)
+    func getDefaultDataMasks() -> [String] {
+        let str = Usabilla.defaultDataMasks
```
Comment (kioli) [inline ios/ReactNativeUsabilla/UsabillaInterface.swift]: Isn't assignment and return in two separate lines a bit verbose with such short line?
Just a personal preference, no need to take action on it

File: ios/ReactNativeUsabilla/UsabillaInterface.swift (hunk 1 above)
Comment (hiteshjain29) [reply ios/ReactNativeUsabilla/UsabillaInterface.swift]: To avoid warning we use _ operator.
As assigning it to unused constant or variable cause warning.

**Pull request merged**